CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+### 1.3.3 / 2019-11-11
+
+* Allow to create vm with multiple pvcs
+* Rename Volume to PersistentVolume
+* Add support for Storage Class
+* Make VmVolume a Fog model
+* Extend VMs, VM Instances and servers
+
```

Comment:
this list doesn't reflect the changes since 1.3.2.
The only recent changes are supporting specific kubevirt version and fix the api version.